siemens/questa.mk

```diff
@@ -158,9 +177,19 @@ $(DEP_DIR)/%.questa.o: $(SIM_LIB_DONE) | $(DEP_DIR) $(BLOG_DIR)
 	else echo "Unknown filetype: $(word 2,$^)"; echo "$^"; exit 1; fi; fi;
 	@touch $@
 
+AC_CMD = qverify -c -do $(AC_SCRIPT) -od $(AC_OUT_DIR) -modelsimini $(MS_INI)
+AC_MSG := Running Autocheck
+$(AC_DONE): $(MS_INI) $(DEP_DIR)/$(_TOP).questa.o $(precomp_hook) $(AC_DIRECTIVES)
+	@printf "$(autocheck_str)" > $(AC_SCRIPT)
+	@echo "Autocheck dependencies changed";
```

Comment:
This `echo` is unnecessary, obviously autocheck is running because dependency analysis said to. There are a couple of similar messages in other parts of `hdl_build` but only when there's no other way the user can tell it happened.